gnarly.rb

```diff
@@ -123,7 +86,7 @@ def setup_rspec
   remove_file ".rspec"
   copy_file "templates/.rspec", ".rspec"
   gsub_file "spec/rails_helper.rb",
-    "# Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }",
+    /# Dir\[Rails\.root\.join.*/,
```

Comment:
Rebase?

Reply:
Yeah, gotta. Had this in because I wasn't sure how long a review on the other PR was going to take and wanted to see if specs pass.